src/test/java/com/artipie/asto/StorageSaveAndLoadTest.java

```diff
@@ -115,6 +115,24 @@ void shouldFailToSaveErrorContent(final Storage storage) throws Exception {
         );
     }
 
+    @TestTemplate
+    @Timeout(1)
+    void shouldFailOnSecondConsumeAttempt(final Storage storage) {
+        Assertions.assertThrows(
+            IllegalStateException.class,
+            () -> {
+                final Key.From key = new Key.From("key");
+                storage.save(
+                    key,
+                    new Content.OneTime(new Content.From("val".getBytes()))
+                ).join();
+                final Content value = storage.value(key).join();
+                Flowable.fromPublisher(value).toList().blockingGet();
+                Flowable.fromPublisher(value).toList().blockingGet();
```

Comment:
@Sammers21 please minimize this failing section to just the last line, so we are sure that the code fails exactly where we expect it to fail